docs/content/reference/cli/authelia/authelia_crypto_rand.md

```diff
@@ -49,6 +51,8 @@ authelia crypto rand --characters 0123456789ABCDEF
   -x, --charset string      sets the charset for the random password, options are 'ascii', 'alphanumeric', 'alphabetic', 'numeric', 'numeric-hex', and 'rfc3986' (default "alphanumeric")
   -h, --help                help for rand
   -n, --length int          sets the character length for the random string (default 72)
+  -q, --quiet               suppress additional output
+  -o, --output-file string  file where the random string will be stored
```

Comment:
I think to stay consistent with other commands, like [`authelia storage user totp export`](https://www.authelia.com/reference/cli/authelia/authelia_storage_user_totp_export/), [`authelia storage user webauthn export`](https://www.authelia.com/reference/cli/authelia/authelia_storage_user_webauthn_export/) and more, we should use `-f`/`--file` as the output file flag.
Especially since `-o` is being used as the "certificate organization" flag in the [`authelia crypto certificate rsa generate`](https://www.authelia.com/reference/cli/authelia/authelia_crypto_certificate_rsa_generate/) command.